src/Hasse_complex/include/gudhi/Hasse_complex.h

```diff
@@ -173,9 +173,9 @@ class Hasse_complex {
   }
 
   void display_simplex(Simplex_handle sh) {
-    std::cout << dimension(sh) << "  ";
-    for (auto sh_b : boundary_simplex_range(sh)) std::cout << sh_b << " ";
-    std::cout << "  " << filtration(sh) << "         key=" << key(sh);
+    std::clog << dimension(sh) << "  ";
```

Comment:
I think (not sure) this one makes more sense in stdout (well, it shouldn't hardcode where it is writing, but that whole class is undocumented anyway, so I don't care about this specific file). The user explicitly asked the program to print this by calling a function called `display_simplex`, this isn't some unnecessary trace. One goal of moving some output to stderr is so that it can easily be separated from the stdout output, not so that stdout disappears. I don't know if there are more interesting examples elsewhere.